datasets/nlcd-urbanization.data.mdx

```diff
@@ -66,32 +106,86 @@ layers:
+  ### Other Publications
 
-  * Rigge, Matthew B., Shi, Hua, Postma, Kory, Projected change in rangeland fractional component cover across the sagebrush biome under climate change through 2085, v. 12, no. 6, at https://doi.org/10.1002/ecs2.3538
+  Danielson, Patrick, Postma, Kory, Riegle, J., Dewitz, Jon A., Deep learning artificial intelligence (AI) for improving classification accuracy for the National Land Cover Database (NLCD) [abs.]
 
-  * Rigge, Matthew B., Homer, Collin G., Shi, Hua, Meyer, Debbie K., Bunde, Brett, Granneman, Brian, Postma, Kory, Danielson, Patrick, Case, Adam, Xian, George Z., Rangeland fractional components across the western United States from 1985 to 2018: Remote Sensing, v. 13, no. 4, at https://doi.org/10.3390/rs13040813
+  Jin, Suming, Dewitz, Jon A., Sorenson, D., Shogib, Rakibul , Granneman, Brian J., Case, Adam, Li, Congcong, Zhe, Z., Danielson, Patrick, Costello, C., Gass, L., National Land Cover Database 2019—A comprehensive strategy for creating the 1986-2019 Forest Disturbance Date Product [abs.], v. Proceedings, at https://agu.confex.com/agu/fm21/meetingapp.cgi/Paper/960755
 
-  * Peng, D., Wang, Y. L., Xian, George, Huete, A.R., Huang, W., Shen, M., Wang, F., Yu, L., Liu, L., Xie, Q., Liu, L., Zhang, X., Investigation of land surface phenology detections in shrublands using multiple scale satellite data: Remote Sensing of Environment, v. 252, at https://doi.org/10.1016/j.rse.2020.112133
+  Rigge, Matthew B., Homer, Collin G., Shi, Hua, Meyer, Debbie K., Bunde, Brett, Granneman, Brian, Postma, Kory, Danielson, Patrick, Case, Adam, Xian, George Z., Rangeland fractional components across the western United States from 1985 to 2018: Remote Sensing, v. 13, no. 4, at https://doi.org/10.3390/rs13040813
 
-  * Rigge, Matthew B., Shi, Hua, Meyer, Debbie K., Bunde, Brett, Postma, Kory, Trends of fractional rangeland components across a 1985-2020 time-series [poster], v. Measuring & Monitoring Ecosystems, at http://annualmeeting2021.rangelands.org/presentations-posters/
+  Wickham, J., Stehman, S.V., Sorenson, D.G., Gass, L., Dewitz, Jon A., Thematic accuracy assessment of the NLCD 2016 land cover for the conterminous United States: Remote Sensing of Environment, v. 257, at https://doi.org/10.1016/j.rse.2021.112357
 
-  * Jin, Suming, Dewitz, Jon A., Sorenson, D., Shogib, Rakibul , Granneman, Brian J., Case, Adam, Li, Congcong, Zhe, Z., Danielson, Patrick, Costello, C., Gass, L., National Land Cover Database 2019—A comprehensive strategy for creating the 1986-2019 Forest Disturbance Date Product [abs.], v. Proceedings, at https://agu.confex.com/agu/fm21/meetingapp.cgi/Paper/960755
+  </Prose>
+</Block>
+
+<Block>
+  <Prose>
+  
+  ## Data Stories Using This Dataset
+  
+  **[Aerosols and Their Impacts on Houston, TX](https://www.earthdata.nasa.gov/dashboard/stories/houston-aod)**
 
```

Comment:
Internal link, use `<Link>` instead

can you find all of such URLs and update them. See example here
https://github.com/US-GHG-Center/veda-config-ghg/pull/572/commits/f91b4041e359d15daea54e78b2fa01f4190c7af6

Reply:
@slesaad I have updated the internal link with <Link> style, but the preview is failing due to an unexpected character error ',' . Is this because the story titles used in the link prose have commas?